CONTRIBUTING.md

```diff
@@ -73,7 +73,7 @@ guidelines:
 
  7. For user visible changes (API changes, behaviour changes, ...),
     consider adding a note in [CHANGES.md](CHANGES.md).
-    This could be a summarising description of the change, and could
+    This could be a summarizing description of the change, and could
```

Comment:
As per our [documentation policy](https://www.openssl.org/policies/technical/documentation-policy.html#language), British English spellings should be used.  This make this change and other similar changes wrong.

Reply:
Solved